maturity-level-one/src/Open-Api-Docs.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>Codit.LevelOne</name>
+    </assembly>
+    <members>
+        <member name="M:Codit.LevelOne.Controllers.ErrorController.Error(System.Int32)">
+            <summary>
+            Manages the unmatched routes
+            </summary>
+            <param name="code">HTTP status code</param>
+            <returns>Error formatted as application/problem+json</returns>
+        </member>
+        <member name="M:Codit.LevelOne.Controllers.HealthController.Get">
+            <summary>
+                Get Health
+            </summary>
+            <remarks>Provides an indication about the health of the runtime</remarks>
+        </member>
+        <member name="T:Codit.LevelOne.Controllers.PlayersController">
+            <summary>
+            All about the football players
+            </summary>
+        </member>
```

Comment:
This is an example of why you should use `Summary` in XML docs for operation summary and remark for more information. This will be awful if you would import it in API Management for example if we use the general approach in this sample over the Get Health one

Reply:
See: https://blog.tomkerkhove.be/2018/04/13/changes-to-azure-api-management-openapi/

Reply:
The current guideline is: 
Summary => short description
Remarks => long description

I do understand that a "long" summary will result in a long "Display Name" in APIM.
We could make clear in the guideline what is the side effect of a long summary (from an APIM point of view).

How would you add to the guideline?

Reply:
Something along the lines of this:
> `Summary` should provide a brief and clear name for the operation where `Remarks` provides a well-written business description of what the operation does and should not leak any technical information.